src/frontends/paddlepaddle/src/input_model.hpp

```diff
@@ -7,28 +7,28 @@
 #include <common/telemetry_extension.hpp>
 #include <manager.hpp>
 
-#include "paddlepaddle_frontend/utility.hpp"
-
 namespace ov {
 namespace frontend {
-class OpPlacePDPD;
-class TensorPlacePDPD;
-class PDPD_API InputModelPDPD : public InputModel {
-    friend class FrontEndPDPD;
+namespace paddlepaddle {
+
+class OpPlace;
+class TensorPlace;
+class FRONTEND_API InputModel : public ov::frontend::InputModel {
```

Comment:
if it's moved to `src`, exporting `FRONTEND_API` is not needed

Reply:
fixed